main/modes/platformer/mode_platformer.c

```diff
@@ -1019,7 +232,10 @@ void platformerEnterMode(display_t *disp)
     platformer->prevBtnState = 0;
 
     loadPlatformerHighScores(platformer);
-    //insertScoreIntoHighScores(&(platformer->highScores), 1000000, "EFV", getHighScoreRank(&(platformer->highScores), 1000000));
+    loadPlatformerUnlockables(platformer);
+    if(platformer->highScores.initials[0][0] == 'E' && platformer->highScores.initials[0][1] == 'F' && platformer->highScores.initials[0][2]){
```

Comment:
Was this supposed to check a third char?

Reply:
Oops, yes. Will fix.